supplementary/alica_ros_turtlesim/libalica-turtlesim/src/WaitForTrigger.cpp

```diff
@@ -0,0 +1,43 @@
+#include "WaitForTrigger.h"
```

Comment:
How is this different than WaitForMsg?

Reply:
WaitForTrigger waits for an empty msg and then succeeds, WaitForMsg waits for a String msg, stores it on the blackboard and then succeeds. I removed it from the lib so we only have it in the ros1 / ros2 folders with their ros specific implementations